<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OpenPort">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Port</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Settings">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Com Port Setting</string>
    </property>
    <widget class="QLabel" name="SerialPort_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial Port</string>
     </property>
    </widget>
    <widget class="QLabel" name="Baudrate_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate(bps)</string>
     </property>
    </widget>
    <widget class="QLabel" name="DataBits_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Bits</string>
     </property>
    </widget>
    <widget class="QLabel" name="StopBits_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Bits</string>
     </property>
    </widget>
    <widget class="QLabel" name="Parity_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Parity</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="AutoConnect">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto connect</string>
     </property>
    </widget>
    <widget class="QComboBox" name="SerialPort">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Baudrate">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>56000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="DataBits">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="StopBits">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="Parity">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Odd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Even</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="ClosePort">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusOfReceivedData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Received Data : Receive 0 characters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rx Format</string>
    </property>
   </widget>
   <widget class="QComboBox" name="RxFormat">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ASCII</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HEX</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>681</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RX Section</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>681</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>651</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TX Section</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusOfSentData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Data: Send 0 characters</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>421</width>
      <height>191</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
   </widget>
   <widget class="QPushButton" name="SendFormat_1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AutoSend_1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AutoSend_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AutoSend_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AutoSend_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AutoSend_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Send_1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Send_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>400</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendFormat_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendFormat_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Send_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendFormat_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Send_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>460</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendFormat_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Send_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SendRepeatly">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Data Repeatedly</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Add_CR_LF">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>520</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add CR/LF</string>
    </property>
   </widget>
   <widget class="QLabel" name="SerialPort_label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FrequencyofSendMsg">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="SerialPort_label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>520</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>/seconds</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSerialCom_QT">
    <property name="title">
     <string>File(F)</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCapture">
    <property name="title">
     <string>Capture</string>
    </property>
    <addaction name="actionStart_Capture_S"/>
    <addaction name="actionStop_Capture_T"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(H)</string>
    </property>
    <addaction name="actionAbout_SerialCom_QT"/>
   </widget>
   <addaction name="menuSerialCom_QT"/>
   <addaction name="menuCapture"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File(O)</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File(S)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit(X)</string>
   </property>
  </action>
  <action name="actionStart_Capture_S">
   <property name="text">
    <string>Start Capture(S)</string>
   </property>
  </action>
  <action name="actionStop_Capture_T">
   <property name="text">
    <string>Stop Capture(T)</string>
   </property>
  </action>
  <action name="actionAbout_SerialCom_QT">
   <property name="text">
    <string>About SerialCom_QT...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
